packages/hub/package.json

```diff
@@ -15,9 +15,9 @@
     "rebuild": "webpack --watch --progress",
     "start": "run-p -c start:*",
     "start:server": "node --no-deprecation -r source-map-support/register dist/hub.js server",
-    "start:worker": "node --no-deprecation -r source-map-support/register dist/hub.js worker",
-    "start:bot": "node --no-deprecation -r source-map-support/register dist/hub.js bot",
-    "start:event-listener": "node --no-deprecation -r source-map-support/register dist/hub.js event-listener",
+    "start:worker": "HEALTH_CHECK_PORT=3001 node --no-deprecation -r source-map-support/register dist/hub.js worker",
+    "start:bot": "HEALTH_CHECK_PORT=3002 node --no-deprecation -r source-map-support/register dist/hub.js bot",
+    "start:event-listener": "HEALTH_CHECK_PORT=3002 node --no-deprecation -r source-map-support/register dist/hub.js event-listener",
```

Comment:
we set different ports for the health checks so they don't collide if you want to run all the things at once